build.sbt

```diff
@@ -36,7 +36,7 @@ onLoad in Global := { s =>
 cancelable in Global := true
 
 // some extra commands for us
-addCommandAlias("genEnums", "; sql/runMain gem.sql.Main modules/core/shared/src/main/scala/gem/enum; headerCreate")
```

Comment:
I think we can leave the compile off and just let that be a separate step.  The headers will be added then just as for everything else.

Reply:
That's fine with me.